frontend/src/pages/chat/libs/constants/constants.ts

```diff
@@ -1,44 +1,40 @@
-import { UserRole } from '#libs/enums/enums.js';
-import { generateUUID } from '#libs/helpers/helpers.js';
-import { type ChatMessage } from '#libs/types/types.js';
+import { type ChatMessageGetAllItemResponseDto } from '#packages/chat-messages/chat-messages.js';
+
+const EMPTY_ARRAY_LENGTH = 0;
```

Comment:
It's common constant, lets store it in `shared/src/libs/constants` folder. At now this approach isn't implemented on development branch, but hopefully it will after some mergings

Reply:
good point! Changed it